website/orders/models.py

```diff
@@ -119,6 +130,22 @@ class Product(models.Model):
         "bound to the venue. Empty means no limit.",
     )
 
+    scannable = models.BooleanField(default=False, help_text="Whether or not this product can be scanned.")
```

Comment:
Can be removed, as a barcode value of Null implies this

Reply:
True